src/collective/mustread/models.py

```diff
@@ -17,10 +17,12 @@ class MustRead(Base):
     userid = Column(Unicode(255), nullable=True)
     read_at = Column(DateTime, nullable=True)
     deadline = Column(DateTime, nullable=True)
+    scheduled_at = Column(DateTime, nullable=True)
+    scheduled_by = Column(String(255))
```

Comment:
good point, i'll add scheduled_by to the signature and will not implicitly use current_user_id